client/components/videos-ui/video-player.jsx

```diff
@@ -1,6 +1,15 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
+
+const VideoPlayer = ( { completedSeconds, videoRef, videoUrl, isPlaying, poster = undefined } ) => {
+	const [ addTimeUpdateHandler, setAddTimeUpdateHandler ] = useState( true );
+
+	const markVideoAsComplete = () => {
+		if ( videoRef.current.currentTime < completedSeconds ) {
+			return;
```

Comment:
I think I think we'll want to check for the existing value of `addTimeUpdateHandler` here as well; otherwise we'll be updating the value a billion times once the user passes the relevant timestamp.

Reply:
Hm, not sure I understand. If we pass the `completedSeconds` value, setting `AddTimeUpdateHandler` to `false` triggers a re-render that skips adding the `onTimeUpdate` handler (and means `markVideoAsComplete` is no longer called). Does that make sense? Or I might be missing something. 🤔 

<img width="666" alt="Screen Shot 2021-11-30 at 11 36 47 AM" src="https://user-images.githubusercontent.com/349751/144115711-ef1b7989-4e29-498c-91ee-4d50b6ce26b9.png">

![2021-11-30 11 36 14](https://user-images.githubusercontent.com/349751/144115753-711f1cf7-79a5-4b2d-86bb-bd4f404b1160.gif)

Reply:
Oh wait, never mind, I misunderstood what you were saying. And yeah, I see what you mean now. Never mind.